models/calendar.yml

```
@@ -157,13 +157,14 @@
 
     - 7:45 PM:
         type: Talk
-        emoji: 💡
-        title: TBD
-        description: TBD
+        emoji: ✨
+        title: Juice Box Driven Design
+        description: A talk about building for mental health: why lightweight apps are so important 
```

```suggestion
        description: |
          A talk about building for mental health: why lightweight apps are so important 
```

I believe the test is breaking due to the `:` in your description. I think using a `|` then a newline will fix the issue!

---

Ah, whoops! 

:pray: sorry about that :pray:

---

No worries! Looking forward to your talk 🎉